.github/workflows/examples_js.yml

```diff
@@ -0,0 +1,92 @@
+name: JS Example Tests
+
+on:
+  workflow_call:
+    inputs:
+      working-directory:
+        required: true
+        type: string
+      api_key:
+        required: false
+        type: string
+      base_url:
+        required: false
+        type: string
+      author:
+        required: false
+        type: string
+      commit_message:
+        required: false
+        type: string
+      dont_notify:
+        required: false
+        type: boolean
+  workflow_dispatch:
+    inputs:
+      working-directory:
+        required: true
+        type: string
+      api_key:
+        required: false
+        type: string
+      base_url:
+        required: false
+        type: string
+      author:
+        required: false
+        type: string
+      commit_message:
+        required: false
+        type: string
+      dont_notify:
+        required: false
+        type: boolean
+
+jobs:
+  run-js-tests:
+    name: Run JS Example Tests
+    defaults:
+      run:
+        working-directory: ${{ inputs.working-directory }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        node-version: ["18"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install Dependencies
+        run: |
+          npm install @ai-sdk/openai ai composio-core dotenv zod
+      - name: Run Tests
+        env:
+          API_KEY: ${{ inputs.api_key || secrets.API_KEY }}
+          BASE_URL: ${{ inputs.base_url || secrets.BASE_URL || 'https://api.example.com' }}
+          DOCKER_USER: ${{secrets.DOCKER_USER}}
+          OPENAI_API_KEY: ${{secrets.OPENAI_API_KEY}}
+          JULEP_API_KEY: ${{secrets.JULEP_API_KEY}}
+          JULEP_API_URL: ${{secrets.JULEP_API_URL}}
```

Comment:
`JULEP_API_URL` Not need. Can be hardcoded.

Reply:
It's getting used for running examples in python, i just added it for future usecases